connector/src/main/java/org/geysermc/connector/entity/FallingBlockEntity.java

```diff
@@ -37,4 +40,12 @@ public FallingBlockEntity(long entityId, long geyserId, EntityType entityType, V
 
         this.metadata.put(EntityData.VARIANT, BlockTranslator.getBedrockBlockId(javaId));
     }
+
+    @Override
+    public void updateBedrockMetadata(EntityMetadata entityMetadata, GeyserSession session) {
+        // Set the NO_AI flag based on the no gravity flag to prevent movement
+        if (entityMetadata.getId() == 5) {
+            this.metadata.getFlags().setFlag(EntityFlag.NO_AI, (boolean)entityMetadata.getValue());

```

Comment:
`(boolean)entityMetadata.getValue()` -> `(boolean) entityMetadata.getValue()` is more consistent with the current Geyser codebase.